Description: Don't build the binary application
Author: Daniele Tricoli <eriol@debian.org>
Forwarded: not-needed
Last-Update: 2022-03-30 <YYYY-MM-DD, last update of the meta-information, optional>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/main.rs
+++ /dev/null
@@ -1,5 +0,0 @@
-extern crate local_ipaddress;
-
-fn main() {
-    println!("{}", local_ipaddress::get().unwrap());
-}
